From 9a9f471bb1aa64b31be40eb81308dd7fe3b2671d Mon Sep 17 00:00:00 2001
From: Oren Givon <oren.givon@intel.com>
Date: Thu, 21 Nov 2013 12:18:51 +0200
Subject: REVERTME Build external iwlwifi modules

Add an option to install extra modules after the external
modules had been installed. This is used to install the
iwlwifi modules.
A better way of adding external modules is needed.

Change-Id: I412a7ed02e224d3d13ec98a55ad26298b6b07536
Signed-off-by: Oren Givon <oren.givon@intel.com>
Signed-off-by: Hakan Englund <hakan.englund@intel.com>
Signed-off-by: Mihai Serban <mihai.serban@intel.com>
---
 core/kernel.mk |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/core/kernel.mk b/core/kernel.mk
index 7f09802..c771675 100644
--- a/core/kernel.mk
+++ b/core/kernel.mk
@@ -164,6 +164,7 @@ endef
 define make-modules
 	$(mk_kernel) INSTALL_MOD_PATH=$(1) modules_install
 	$(foreach item,$(dir $(EXTERNAL_KERNEL_MODULES_TO_INSTALL)),$(call make-ext-module,$(item),$(1)))
+	$(foreach item,$(EXTRA_KERNEL_MODULES), make $(item))
 	$(hide) rm -f $(1)/lib/modules/*/{build,source}
 	$(hide) cd $(1)/lib/modules && find -type f -print0 | xargs -t -0 -I{} mv {} .
 endef
-- 
1.7.4.1

